pyproject.toml

```diff
@@ -19,7 +19,7 @@ dependencies = [
     "Click",
     "GeoAlchemy2>=0.9,!=0.11.*",
     "SQLAlchemy>=1.4",
-    "threedi-schema==0.220.*"
+    "threedi-schema@https://github.com/nens/threedi-schema/archive/refs/heads/margriet_46_schema_300.zip"
```

Comment:
Note: this needs to be removed/changed after threedi-schema has been released.

Reply:
fixed